debian/mariadb-server-10.5.install

```diff
@@ -5,6 +5,7 @@ debian/additions/mariadb.conf.d/50-mysqld_safe.cnf etc/mysql/mariadb.conf.d
 debian/additions/mariadb.conf.d/50-server.cnf etc/mysql/mariadb.conf.d
 debian/additions/source_mariadb-10.5.py usr/share/apport/package-hooks
 etc/apparmor.d/usr.sbin.mariadbd
+etc/mysql/mariadb.conf.d/spider.cnf
```

Comment:
is there such a file? where is it generated?

Reply:
It is in sources `storage/spider/spider.cnf` and installed by CMake automatically I guess?

Reply:
generated from cmake/plugin.cmake